AsyncDisplayKit/ASDisplayNode+Subclasses.h

```diff
@@ -324,21 +324,21 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, readonly, assign, getter=isInHierarchy) BOOL inHierarchy;
 
 /**
- * @abstract Indicates that the node should fetch any external data, such as images.
+ * @abstract Indicates that the node should preload any external data, such as images.
  *
- * @discussion Subclasses may override this method to be notified when they should begin to fetch data. Fetching
+ * @discussion Subclasses may override this method to be notified when they should begin to preload. Fetching
  * should be done asynchronously. The node is also responsible for managing the memory of any data.
  * The data may be remote and accessed via the network, but could also be a local database query.
  */
-- (void)fetchData ASDISPLAYNODE_REQUIRES_SUPER;
+- (void)preload ASDISPLAYNODE_REQUIRES_SUPER;
```

Comment:
This one is trickier than I thought -- because of setNeedsDataFetch.  I think we should probably eliminate this method for now, and just call didEnterPreloadState again (even if imbalanced) when setNeedsDataFetch is called.  Let's deprecate setNeedsDataFetch and circle back to a cleaner API for "re-fetching" when we have more time (and we'll add that better API before finalizing removal of the deprecated method).